frontend/src/metabase/components/DateRelativeWidget/DateRelativeWidget.module.css

```diff
@@ -1,6 +1,13 @@
-.shortcut:hover {
+.shortcutSelected {
   color: var(--mb-color-text-selected);
   background-color: var(--mb-color-background-selected);
+  border-color: var(--mb-color-background-selected);
```

Comment:
Also, modify the border-color so it doesn't look different than other filled button.